### PR TITLE
New #25

### PR DESCRIPTION
**Pull request by yowko**
Add the test adapter as part of the package

### CODE REVIEW

**Comment by ChrisMaddock** [comment]
Nice idea! 😄 

I did something a bit like this in https://github.com/nunit/nunit-vs-testgenerator/pull/17 - @OsirisTerje found out that if you pass `null` in place of the version, it will always get the latest NuGet package. That would save a maintenance cost here - maybe an improvement? 🙂

**Comment by OsirisTerje** [comment]
@ChrisMaddock Yep, that is right.  So the same thing should be done here I think.  Also, since we're already on 3.8 too.  @yowko Can you do that?  
(I see the entry for NUnit 2 also has a version string, we need to keep that, otherwise we will get the NUnit 3 latest version, the adapter for 2.X however, can be null-versioned.)

Just for info:  We didn't add the adapter earlier because you really only need one in one of the projects, but I guess it doesn't matter if they are duplicated as entries, there will still only be one download.

**Comment by yowko** [comment]
@OsirisTerje Am I right?

**Review by OsirisTerje — COMMENTED** [review]
Yep, one more though :-)  See code comment

File: CreateUnitTests.NUnit/NUnit2SolutionManager.cs [hunk 1]
```diff
@@ -65,6 +66,7 @@ protected override void OnUnitTestProjectCreated(Project unitTestProject, CodeFu
 
             base.OnUnitTestProjectCreated(unitTestProject, sourceMethod);
             EnsureNuGetReference(unitTestProject, "NUnit", NUnitVersion);
+            EnsureNuGetReference(unitTestProject, "NUnitTestAdapter", NUnitTestAdapterVersion);
```
Comment (OsirisTerje) [inline CreateUnitTests.NUnit/NUnit2SolutionManager.cs]: This one (for NUnitTestAdapter) can also be set to null.

**Review by OsirisTerje — APPROVED** [review]

**Pull request merged**